tools/hardhat-example/scripts/transferHbars.js

```diff
@@ -0,0 +1,15 @@
+const hre = require('hardhat');
+
+module.exports = async (amount = 100000000000) => {
```

Comment:
nit: for readability
```suggestion
module.exports = async (amount = 100_000_000_000) => {
```